spec/terminology.md

```diff
@@ -3,17 +3,74 @@
+~ A verifiable append-only log of key events for an [[ref: AID]] that is both backward and forward-chained.
 
-[[def: transaction event log, TEL]]
-~ As defined by KERI.
+[[def: key event receipt]]
+~ A message whose body references a [[ref: key event] of an [[ref: AID]] and includes one or more signatures on that key event.
+
+[[def: key event receipt log (KERL), KERL]]
+~ A verifiable append-only log that includes all the consistent [[ref: key event receipt]] messages created by [[ref: witness]]es.
+
+[[def: key event receipt infrastructure (KERI), KERI]]
+~ A protocol that provides an identity system-based secure overlay for the internet. The KERI protocol includes a primary root-of- trust in self-certifying identifiers (SCIDs), called [[ref: autonomic identifier]]s ([[ref: AID]]s), that are strongly bound to an append-only chained [[ref: key event log]] ([[ref: KEL]]) of signed transfer statements provides end verifiable control provenance.
 
 [[def: KERI event stream, KERI event streams]]
 ~ A stream of verifiable KERI data, consisting of the [[ref: key event log]] ([[ref: KEL]]) and other data such as [[ref: transaction event log]] ([[ref: TEL]]). This data is a [[ref: CESR]] event stream (TODO: link to IANA application/cesr media type) and may be serialized in a file using [[ref: CESR]] encoding. We refer to these CESR stream resources as KERI event streams to simplify the vocabulary.
+
+[[def: key state]]
+~ The set of currently authoritative keypairs (current keys) for an [[ref: AID]] and any other information necessary to secure or establish control authority over the AID.
+
+[[def: method-specific identifier, MSI]]
+~ The `method-specific-id` part of DID Syntax, as defined in [DID Core](https://www.w3.org/TR/did-core/#did-syntax). See section [Method-Specific Identifier](#method-specific-identifier).
+
+[[def: out-of-band introduction (OOBI), OOBI]]
+~ A protocol for discovering verifiable information on an [[ref: AID]] or a [[ref: SAID]]. The OOBI by itself is insecure, and the information discovered by the OOBI must be verified.
+
+[[def: pre-rotation]]
+~ A key rotation mechanism whereby a set of rotation keys are pre-commited using cryptographic digests. The pre-rotation mechanism enables an entity to persistently maintain or regain control over an [[ref: AID]] in spite of the exposure-related weakening over time or even compromise of the current set of controlling (signing) keypairs.
+
+[[def: rotation event]]
+~ A key establishment event that provides the information needed to change the [[ref: key state]] which includes a change to the set of authoritative key pairs for an [[ref: AID]].
+
+[[def: self-addressing identifier (SAID), SAID]]
+~ An identifier that is content-addressable and self-referential. A SAID is uniquely and cryptographically bound to a serialization of data that includes the SAID as a component (or field) in that serialization.
+
+[[def: transaction event log, TEL]]
+~ As defined by KERI.
```

Comment:
Isn’t there more that we can provide about this to help a `did:webs` specification reader?  This is a pretty important concept, as it is what makes verifiable the events other than key events.

Reply:
yes, see my remark here: https://github.com/trustoverip/tswg-did-method-webs-specification/pull/66#discussion_r1364355672

Reply:
How about "A verifiable append-only log of transaction data that are cryptographically anchored to a [[ref: KEL]]. The transaction events of a TEL may be used to establish the issuance or revocation state of [[ref: ACDC]]s."?